.github/ISSUE_TEMPLATE/new_product_suggestion.md

```diff
@@ -0,0 +1,26 @@
+---
+name: New Product Suggestion
+about: Suggest a new product to add into endoflife.date
+title: ''
+labels: 'new-product'
+assignees: ''
+
+---
+
+`Full and Short Name of Suggested Product`
```

Comment:
```suggestion
**`**Full and short name of product**
```

Reply:
you made a mistake while editing ( forgotten ` ) , fixed but in general form i applied what you suggested

Reply:
Oops, thank you!